src/Typesense/Errors/ImportError.ts

```diff
@@ -1,8 +1,8 @@
 import TypesenseError from "./TypesenseError";
-import { ImportResponseFail } from "../Documents";
+import { ImportResponse } from "../Documents";
 
 export default class ImportError extends TypesenseError {
-  importResults: ImportResponseFail;
+  importResults: ImportResponse[];
   constructor(message, importResults) {
```

Comment:
```suggestion
  constructor(message: string, importResults: ImportResponse[]) {
```
Good point, thanks @tharropoulos - is this what you were thinking? Will add to the PR.

Reply:
Exactly! Thank you!